package.json

```diff
@@ -7,6 +7,7 @@
     },
     "scripts": {
         "dev": "next dev",
+        "dev:https": "next dev --experimental-https",
```

Comment:
q: what's this script for? Can we add a description in the README or somewhere?

Reply:
There are some features of pwa and services workers that require being served over https, for example the issue I had with an event not firing right was only happening over https. Also testing out PUSH (removed for now) requires https. These cases come up here and there actually, just thought I'd make it available in the scripts.
Adding to README